taskcluster/scripts/detect-target.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
```

Comment:
Should this script crash in the event of an error? i.e. should we `set -euo pipefail` ?